edx-solutions/xblock-drag-and-drop-v2
# Mode field

**Description:** This PR adds a mode field to DnDv2 with to possible values: "standard" and "assessment" + studio editing capabilities.

**Dependencies:** Based on https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/74

**Sandbox LMS URL**: http://dndv2.opencraft.hosting/
**Sandbox Studio URL**: http://studio-dndv2.opencraft.hosting/

**Testing instructions:**
1. Install XBlock into LMS/Studio
2. Enable in Advanced settings
3. Create a course, section, subsection and unit
4. Add DnDv2 block to a unit
5. Click edit. *Expected result*: Editor popup opens; "Problem mode" select with two values "standard" and "assessment" are present, with "standard" selected
6. Change "Problem mode", save DnDv2 (note the edit dialog have three steps)
7. Click "Edit" again. *Expected result*: Editor popup opens, "Problem mode" is present and previously saved value is selected.

Reviewers:
